src/components/bulkConcepts/component/ActionButtons.jsx

```diff
@@ -35,8 +35,8 @@ export class ActionButtons extends Component {
     this.setState({ open: false });
   };
 
-  fetchPreview = () => {
-    this.props.previewConcept(this.props.id);
+  fetchPreview = (id) => {
```

Comment:
Is this also about testing the untested areas?

Reply:
Yes, this is part of the first bullet point that handles restructuring of the code in order for it to be tested. The function was formally untested and after the change, I was able to test it.